src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -31,8 +31,6 @@ export class ExerciseDetailsStudentActionsComponent implements OnInit {
     readonly QUIZ_NOT_STARTED = ParticipationStatus.QUIZ_NOT_STARTED;
     readonly QUIZ_NOT_PARTICIPATED = ParticipationStatus.QUIZ_NOT_PARTICIPATED;
     readonly QUIZ_FINISHED = ParticipationStatus.QUIZ_FINISHED;
-    readonly MODELING_EXERCISE = ParticipationStatus.MODELING_EXERCISE;
-    readonly TEXT_EXERCISE = ParticipationStatus.TEXT_EXERCISE;
     readonly FILE_UPLOAD_EXERCISE = ParticipationStatus.FILE_UPLOAD_EXERCISE;
```

Comment:
Please here also remove `FILE_UPLOAD_EXERCISE` constant and treat it the same way as modeling and text exercises